src/PlugKeyRing/plugKeyRing.test.ts

```diff
@@ -26,14 +32,53 @@ jest.mock('../utils/dfx', () => {
   };
 });
 
-jest.mock('../utils/dfx/token', () => {
-  return {
-    createTokenActor: (): { getMetadata: jest.Mock<any, any> } => ({
-      getMetadata: jest.fn(() => ({
-        fungible: { symbol: 'WTC', decimals: 5, name: 'Wrapped Cycles' },
-      })),
-    }),
-  };
+const mockedSendToken = jest.fn(({ amount }: any) => ({
```

Comment:
can we move all mocks to their own file by any chance?

Reply:
Not right now. We gonna do a big refactor in the near future and change all that kind of things